app/controllers/ops_controller/settings/common.rb

```diff
@@ -468,7 +468,8 @@ def settings_update_save
 
       save_ntp_server_settings
 
-      if @update.validate                                           # Have VMDB class validate the settings
+      config_valid, config_errors = VMDB::Config::Validator.new(@update).validate
```

Comment:
The validator internally only supports "config" key to support Vmdb::Config...it can aslo take a Settings object or a Settings.to_hash object.  So, if this method can pass in that instead, we can drop Vmdb::Config support from the validator.

Reply:
Alternately, I think this line can be `Vmdb::Settings.validate(@update) # or @update.config`

Reply:
Once we drop Vmdb::Config support from the validator, then the Vmdb::Config::Validator and Vmdb::Config::Activator can be renamed to Vmdb::Settings::*.  Then, there are just a handful of places that use Vmdb::Config in core that are easily switched to Vmdb::Settings.

Reply:
Just saw this, :point_up: sounds rather similar to [my suggestion in the core PR](https://github.com/ManageIQ/manageiq/pull/17970#discussion_r218433959). Glad we agree :smile: